java/.ci/Jenkinsfile

```diff
@@ -719,6 +719,7 @@ pipeline {
                                 --templateProperty project.id=$GCP_PROJECT \
                                 --templateProperty bigquery.gcs.input.table=$GCP_PROJECT:dataproc_templates.emp_table \
                                 --templateProperty bigquery.gcs.output.format=avro \
+                                --templateProperty bigquery.gcs.output.partition.col=emp_city \
```

Comment:
Can you please add one more automation / jenkins test which is without partitions

Reply:
There are two test cases.

1. BIGQUERY TO GCS (avro) which is with partition column emp_city
2. BIGQUERY TO GCS (csv) which is without partition column